GX Password/App.xaml.cs

```diff
@@ -59,6 +62,12 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
                 }
                 Window.Current.Activate();
             }
+
+
+            CoreApplication.GetCurrentView().TitleBar.ExtendViewIntoTitleBar = true;
+            ApplicationViewTitleBar titleBar = ApplicationView.GetForCurrentView().TitleBar;

```

Comment:
Conflict with MainPage.xaml.cs
https://github.com/TiagoDanin/GX-Password/blob/95f95aa7eaaea9422d7703be84c3e6a68d5301b3/GX%20Password/MainPage.xaml.cs#L49

Reply:
Recommend move to MainPage.xaml.cs

Reply:
Ok, I know how to fix the conflict.